app/controllers/admin/api_docs/services_controller.rb

```diff
@@ -101,11 +100,19 @@ def api_docs_params(*extra_params)
     params.require(:api_docs_service).permit(*permit_params)
   end
 
+  def set_service_navigation_if_service
```

Comment:
prefixes `get_` and `set_` should be avoided.

Reply:
what do you think about `show_service_navigation_if_service` @macejmic ?

Reply:
`show` should also be avoided as it leads one to think it is view-related stuff added to the controller. And, honestly, I think `_if_service` is worse than `set_ `.

Reply:
what about `toggle_service_navigation, if: bla_bla`

Reply:
@guicassolato the `if` is clear to me from https://github.com/3scale/porta/pull/72#discussion_r223358337 😉 
The beginning of the method is what is not clear to me.
`toggle` is not what is does 😝 
maybe `activate_service_navigation` then? 😄

Reply:
`activate_` sounds pretty cool to me 🍤